src/interface/src/app/password-reset/confirmation-dialog/confirmation-dialog.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-confirmation-dialog',
+  templateUrl: './confirmation-dialog.component.html',
+})
+export class ConfirmationDialogComponent {
+  protected readonly checkUrl = '/assets/png/gm_done_gm_grey_24dp.png';
```

Comment:
nit, maybe this should be called `checkImageUrl` or something that makes clear that its an asset and not a link/route

Reply:
Done.